cddl_gen/cddl_gen.py

```diff
@@ -2299,13 +2303,16 @@ def parse_args():
     code_parser.add_argument(
         "-t", "--entry-types", required=True, type=str, nargs="+",
         help="Names of the types which should have their xcode functions exposed.")
-    code_parser.add_argument(
-        "--time-header", required=False, action="store_true", default=False,
-        help="Put the current time in a comment in the generated files.")
     code_parser.add_argument(
         "-d", "--decode", required=False, action="store_true", default=False, help="Generate decoding code.")
     code_parser.add_argument(
         "-e", "--encode", required=False, action="store_true", default=False, help="Generate encoding code.")
+    code_parser.add_argument(
+        "--time-header", required=False, action="store_true", default=False,
```

Comment:
I'm curious, would you get the same behavior without `required=False` and `default=False`? If that is the case then we could simplify some of these arguments. For now I would say that sticking with the convention is better.

Reply:
Looks like you're right. I'll change it in another PR.